.github/PULL_REQUEST_TEMPLATE/documentation_template.md

```diff
@@ -0,0 +1,76 @@
+# Documentatioon 
+
+**Title:** 
+
+## Category
+
+Which category does this documentation add/updated/remove?
+- [ ] code documentation
+- [ ] wiki
+- [ ] README
+- [ ] installation guide
+    - [ ] OS specifics
+    - [ ] dependency
+    - [ ] environment 
+- [ ] tutorials 
+    - [ ] plotting
+    - [ ] io
+    - [ ] utils/exceptions
+- [ ] examples
+- [ ] citing/licensing
+- [ ] trouble shooting
+    - [ ] installation
+    - [ ] tutorials
+- [ ] history
+- [ ] guidelines 
+    - [ ] developer
+    - [ ] communication
+
+### Details
+
+*Please provide more details on the add/updated/removed documentation on the selected categories.* 
+
+### Goals
+What goals are you trying to achieve with the add/updated/removed documentation? 
+- [ ] readability (fixing typos/grammar)
+- [ ] conciseness (making it less confusing)
+- [ ] updating (updating old plots/code examples)
+- [ ] warnings (warning of caveats or potential issues)
+- [ ] user-interface (making something more user friendly)
+
+## Issue reference
+
+**Issue:** *issue number*
+
+## Checklist
+
+- [ ] spell checker
+- [ ] grammar checker
+- [ ] meets PEP8 documentation style (if applicable)
+- [ ] meets numpy/pandas docstring style (if applicable)
+- [ ] view in readthedocs/markdown viewer (if applicable)
+- [ ] proof read
+
+### Reviewers Help
+
+What do you want the review to focus on? 
+- [ ] spelling/grammar
+- [ ] accuracy
+- [ ] readability
+- [ ] fluiditiy 
+- [ ] test out examples (if applicable)
+- [ ] follow steps (if applicable)
+
+## **readthedocs** Link/Display
+
+*Please provide the readthedocs link to your documentation or how to view it.* 
+
+## Extra notes
+
+*Please provide any other information on this PR here.*
+
+Example info: 
+- does this need to be updated on readthedocs
+- does it require a change in the `setup.py`
+- are there performance hits/improvements?
+- what is the design pattern? 
```

Comment:
An example of what you mean by design pattern, similar to what you provided for the goals, would be helpful here.

Reply:
fixed it in the feature one to point to a link but removed it in the documentation since this is a copy over mistake.